README.md

```diff
@@ -57,6 +57,11 @@
 - [Zsh](http://www.zsh.org/) -  A powerful command line shell. [![Open-Source Software][OSS Icon]](http://sourceforge.net/p/zsh/code/ci/master/tree/)
 
 
+### E-Book Utilities
+
+- [Calibre](http://calibre-ebook.com/) - Incredibly ugly but powerful software for ebook management and conversion. [![Open-Source Software][OSS Icon]](https://github.com/kovidgoyal/calibre)
+
```

Comment:
Looks like you have removed one of the items, earlier it had 2 items if I'm not wrong?